src/daemon/led_mouse_bragi.c

```diff
@@ -24,6 +24,20 @@ int updatergb_mouse_bragi(usbdevice* kb, int force){
 
     uchar pkt[BRAGI_JUMBO_SIZE] = {0};
 
+    if (kb->vendor == V_CORSAIR && kb->product == P_KATAR_PRO_XT) {
```

Comment:
ckb-next does not currently handle hardware brightness; It is implemented in software.
Hardware brightness needs to be set to 100%.

I'd suggest changing this so that it sets it to full brightness on all bragi devices.

Reply:
ok, ive set it to 1000 fixed, ive added a commit on top of that, which you may want to ommit but being able to switch the hw brightness off is useful